packages/astro/package.json

```diff
@@ -54,7 +54,7 @@
     "dev": "astro-scripts dev \"src/**/*.ts\"",
     "postbuild": "astro-scripts copy \"src/**/*.astro\"",
     "benchmark": "node test/benchmark/dev.bench.js && node test/benchmark/build.bench.js",
-    "test": "mocha --parallel --timeout 20000 --ignore **/lit-element.test.js && mocha --timeout 20000 **/lit-element.test.js",
+    "test": "mocha --exit --timeout 20000 --ignore **/lit-element.test.js && mocha --timeout 20000 **/lit-element.test.js",
```

Comment:
This is the fix!

1. Remove `--parallel` - Our tests all default to the default `dist`, which ends up being the output directory of the fixture. If you run multiple test suites in parallel, you'll get conflicts as different tests are touching the same dist folder (building, clearing, previewing, etc.)
2. Add `--exit` - For some reason, CI fails to exit when its moved out of parallel mode due to some resource still being left open. According to docs this is a more "pure" way to run tests, so my guess is that `--parallel` mode exiting was papering over the issue and we actually have a problem with some test leaving resources open. 

Some longer-term action items:
1. We may be able to get back to parallel mode by defaulting our fixtures to creating temp output directories, managing cleanup automatically, etc. This is out of the scope of this PR, and left as a TODO (not sure if we even want to go back to parallel mode)
2. Diagnose why mocha won't exit on its own. If anyone can reproduce the hang locally, this could probably be pretty quick to debug by deleting test files until you can no longer reproduce. According to mocha docs, `--exit` is a fine short-term solution to this problem.

Reply:
Update: real fix is below, so I reverted this one. Still happy to consider it if there's interest